core/src/main/java/com/novoda/noplayer/drm/provision/HttpPoster.java

```diff
@@ -6,7 +6,7 @@
 import java.net.HttpURLConnection;
 import java.net.URL;
 
-class HttpPoster {
+public class HttpPoster {
```

Comment:
these classes were being used in the client project, we'll replace this class with an interface and make this impl package again